lib/sql_tracker.rb

```diff
@@ -8,11 +8,22 @@ def self.initialize!
 
     config = SqlTracker::Config.apply_defaults
     handler = SqlTracker::Handler.new(config)
-    ActiveSupport::Notifications.subscribe('sql.active_record', handler)
+    handler.subscribe
     @already_initialized = true
 
     at_exit { handler.save }
   end
+
+  def self.track
```

Comment:
Would you be able to come out a test for this?

Reply:
Added a test